src/models/user.js

```diff
@@ -2,10 +2,37 @@ const mongoose = require('mongoose');
 
 const Schema = mongoose.Schema;
 
+var subjectSchema = new Schema({
+    subjectId: String,
```

Comment:
We don't want a subjectId, that will be generated by mongodb. What we want is a type or name value. Subjects will exist in a collection that defines their name, id. And then a user has subjects with added price and enabled boolean.